tests/model/test_project_issue.py

```diff
@@ -0,0 +1,124 @@
+#
+# Copyright 2024 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from living_documentation_generator.model.github_project import GithubProject
+from living_documentation_generator.model.project_issue import ProjectIssue
+
+
+# loads
+
+
+def test_loads_with_valid_input():
+    project_issue = ProjectIssue()
+    issue_json = {
+        "content": {"number": 1, "repository": {"owner": {"login": "organizationABC"}, "name": "repoABC"}},
+        "fieldValues": {
+            "nodes": [
+                {"__typename": "ProjectV2ItemFieldSingleSelectValue", "name": "Status1"},
+                {"__typename": "ProjectV2ItemFieldSingleSelectValue", "name": "Priority1"},
+                {"__typename": "ProjectV2ItemFieldSingleSelectValue", "name": "Size1"},
+                {"__typename": "ProjectV2ItemFieldSingleSelectValue", "name": "MoSCoW1"},
+            ]
+        },
+    }
+    project = GithubProject()
+    project.title = "Test Project"
+    project.field_options = {
+        "Status": ["Status1", "Status2"],
+        "Priority": ["Priority1", "Priority2"],
+        "Size": ["Size1", "Size2"],
+        "MoSCoW": ["MoSCoW1", "MoSCoW2"],
+    }
+
+    actual = project_issue.loads(issue_json, project)
+
+    assert actual is not None
+    assert actual.number == issue_json["content"]["number"]
+    assert actual.organization_name == issue_json["content"]["repository"]["owner"]["login"]
```

Comment:
Is this correct? login.name != orga name in GH. But maybe in Project it is different.

Reply:
I looked again in the issue_json structure for project_issue and this should be correct. In the structure you can see the name of the the repository and after the owner of the repository behind the login value.
```
                     nodes {{
                          content {{
                              ... on Issue {{
                                title
                                state
                                number
                                repository {{
                                  name
                                  owner {{
                                    **login**
                                  }}
                                }}
                              }}
                            }}
```